yewprint-doc/src/icon/mod.rs

```diff
@@ -179,7 +183,7 @@ crate::build_example_prop_component! {
                                 ..props
                             })}
                             value_label={
-                                Cow::Owned(format!("{}", ctx.props().example_props.icon_size))
+                                IString::from(format!("{}", ctx.props().example_props.icon_size))
```

Comment:
Already converted for you. That's the whole point of using AttrValue in props
```suggestion
                                format!("{}", ctx.props().example_props.icon_size)
```